iconrpcserver/utils/__init__.py

```diff
@@ -13,7 +13,16 @@
 # limitations under the License.
 
 
-def to_low_camel_case(snake_str: str) -> str:
-    str_array = snake_str.split('_')
-    tmp_str = str_array[0].swapcase() + ''.join(sub.title() for sub in str_array[1:])
-    return tmp_str
+def upper_snake_to_lower_camel(upper_snake: str) -> str:
+    str_array = upper_snake.split('_')
+    return str_array[0].swapcase() + ''.join(sub.title() for sub in str_array[1:])
```

Comment:
if `upper_snake` is `Redirect_PROTOCOL`, then what do you expect result which `upper_snake_to_lower_camel` return?

I suggest to show example on head of function like below.
`request : "REDIRECT_PROTOCOL" -> result : "redirectProtocol"`

Reply:
Thanks for advise! I updated those methods so that it could be used more universally.